rust/src/client/mod.rs

```diff
@@ -1,3 +1,23 @@
+//! Provides client-side functionality to connect to a XayNet service.
+//!
+//! This functionality includes:
+//!
+//! * Abiding by (the underlying [`Participant`]'s side of) the PET protocol.
+//! * Handling the network communication with the XayNet service, including
+//!   polling of service data.
+//!
+//! A [`Client`] has an intentionally simple API - the idea is that it is
+//! initialised with some settings, and then [`start()`]ed. Currently for
```

Comment:
``[`start()`]`` results in a dead link (the rustdoc auto linker has some issue yet in resolving method links). this should easily be fixed by adding the following line to the end of this documentation block:
```rust
//! [`start()`]: #method.start
```